cmd/stolonctl/upgrade.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upgradeCmd = &cobra.Command{
+		Use:   "upgrade",
+		Short: "Upgrade PostgreSQL major version",
```

Comment:
I think I already wondered if this just updated the version of the server or the software itself?